### PR TITLE
fix: 🐛 Fix Button text not align center in Chrome

### LINKED ISSUE #19972: Inconsistent vertical text position in <Button />
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/delicate-lake-hk14y)

### Steps to reproduce
Render multiple `<Button />` components on top of eachother 

### What is expected?
Consistent vertical placement of the text inside the `<Button />` component.

### What is actually happening?
In every other `<Button />` component the text seems to be one pixel closer to the bottom and not vertically centered.

| Environment | Info |
|---|---|
| antd | 3.24.0 |
| React | "react": "^16.9.0" |
| System | macOS Catalina v 10.15 |
| Browser | Chrome Version 77.0.3865.90 (Official Build) (64-bit) |

---
This also happens when you have icons inside the button.

### LINKED ISSUE #20058: Button 组件中文垂直居中稍微偏下
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/antd-reproduction-template-pocpt)

### Steps to reproduce
放一个button控件

### What is expected?
按钮内中文垂直居中

### What is actually happening?
按钮内中文垂直会偏下

| Environment | Info |
|---|---|
| antd | 3.26.0 |
| React | 16.12.0 |
| System | mac 10.14.6 |
| Browser | chrome 78 |

---
发现
https://github.com/ant-design/ant-design/pull/18139/commits/05181d552e452dc5de34e4293abf009be1e89fd3
提交将之前1.4999改成了1.5.我本地覆盖line-height为1.4999后，居中正常
53626D2C-222D-45C7-8981-05FC34DCC2C3
深色为antd:3.26.0
半透明为 antd:3.10.10
文字大概偏下了半像素

### PR DESCRIPTION
**Pull request by afc163**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

revert solution of 05181d552e452dc5de34e4293abf009be1e89fd3

close #19972
close #20058

### 💡 Background and solution

排查了下面所有问题，都能对齐。

- [x] https://github.com/ant-design/ant-design/issues/12978
- [x] https://github.com/ant-design/ant-design/issues/20058
- [x] https://github.com/ant-design/ant-design/issues/19972
- [x] https://github.com/ant-design/ant-design/issues/12978
- [x] https://github.com/ant-design/ant-design/issues/18107
- [x] https://github.com/ant-design/ant-design/issues/13214

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Fix Button text not align well in Chrome.  |
| 🇨🇳 Chinese | 修复 Chrome 下 Button 文字没有垂直居中的问题。 |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/button/style/index.less [hunk 1]
```diff
@@ -73,6 +81,10 @@
 
   &-icon-only {
     .btn-square(@btn-prefix-cls);
+
+    > i {
+      vertical-align: middle;
+    }
```
Comment (afc163) [inline components/button/style/index.less]: icon 改大到 16px 之后按 baseline 对齐就会有问题。

**Comment by afc163** [comment]
效果看上去很完美。

File: components/button/style/index.less (hunk 1 above)
Comment (ztplz) [reply components/button/style/index.less]: 没意义的  解决这些问题又会产生其他新的问题  不同浏览器的处理是有差异的

Comment (afc163) [reply components/button/style/index.less]: 可以帮忙看看。

**Comment by Satloff** [comment]
I think this reintroduces the bug that #18139 fixed. Button text is once again uncentered, especially for larger button size:
Screen Shot 2019-12-15 at 9 59 28 PM
Screen Shot 2019-12-15 at 9 59 14 PM

**Pull request merged**